cli/ballerina-cli/src/main/resources/cli-help/ballerina-encrypt.help

```diff
@@ -6,8 +6,8 @@ SYNOPSIS
 
 
 DESCRIPTION
-       Encrypt command performs symmetric encryption of sensitive
-       configuration values using AES, CBC mode with PKCS#5 padding.
+       Perform symmetric encryption of sensitive
```

Comment:
```suggestion
       Perform AES symmetric encryption of sensitive
```

Reply:
This is not valid since we are removing the `ballerina-encrypt.help` file.